include/sparrow/data_type.hpp

```diff
@@ -129,6 +130,7 @@ namespace sparrow
         float32_t,
         float64_t,
         std::string,
+        std::string_view,
```

Comment:
Same remark as for `char`.

Actually, these are not "base types", this list is only for the types used by default for the related type ids.

If you wanted to add these so that they are automatically tested, I suggest making another test-specific list which would be used in test, or one that is based on all type traits defined in data_traits.hpp?

Reply:
Ok I created another concept (is_arrow_base_type_extended)